Cake.Recipe/Content/build.cake

```diff
@@ -19,38 +19,31 @@ public bool IsSupportedCakeVersion(string supportedVersion, string currentVersio
 // SETUP / TEARDOWN
 ///////////////////////////////////////////////////////////////////////////////
 
-Setup<BuildData>(context =>
+#if !CUSTOM_VERSIONING
+Setup<BuildVersion>(context =>
 {
-    Information(Figlet(BuildParameters.Title));
-
-    Information("Starting Setup...");
-
-    if (BuildParameters.BranchType == BranchType.Master && (context.Log.Verbosity != Verbosity.Diagnostic)) {
-        Information("Increasing verbosity to diagnostic.");
-        context.Log.Verbosity = Verbosity.Diagnostic;
-    }
+    var buildVersion = null;
 
     RequireTool(BuildParameters.IsDotNetCoreBuild ? ToolSettings.GitVersionGlobalTool : ToolSettings.GitVersionTool, () => {
-        BuildParameters.SetBuildVersion(
-            BuildVersion.CalculatingSemanticVersion(
+        buildVersion = BuildVersion.CalculatingSemanticVersion(
                 context: Context
             )
```

Comment:
Looks like you forgot to remove this parenthesis after removing `BuildParameters.SetBuildVersion`